### ISSUE TITLE
Generic.Functions.CallTimePassByReference deprecation notice with PHP_CodeSniffer 3.13.0

### ISSUE THREAD

**Issue by mikemanger** [WordPress/WordPress-Coding-Standards#2528]
Using PHP_CodeSniffer 3.13.0 gives this warning:
```
-  Generic.Functions.CallTimePassByReference
   This sniff has been deprecated since v3.12.1 and will be removed in v4.0.0.
```

It looks like the functionality could be moved over to [PHPCompatibility/PHPCompatibilityWP](https://github.com/PHPCompatibility/PHPCompatibilityWP) by using `PHPCompatibility.Syntax.ForbiddenCallTimePassByReference`.

More info here https://github.com/PHPCSStandards/PHP_CodeSniffer/issues/921

**Comment by edpittol** [comment]
@mikemanger It is possible to exclude the sniff from WordPress rules in your own `phpcs.xml` file. The warning will not be shown. But, of course, the sniff will be removed from the validation.

```xml
<rule ref="WordPress">
    <exclude name="Generic.Functions.CallTimePassByReference" />
</rule>
```
